src/sdk/model/company.ts

```diff
@@ -177,4 +177,32 @@ export class Company extends Entity {
     });
   }
 
+  async setLogo(logo: Uint8Array): Promise<boolean> {
+    return Iland.getHttp().post(`/companies/${this.uuid}/logo`, logo,{
+        headers: {
+            'Content-Type': 'image/jpeg'
+        }
+    }).then(() => {
+        return true;
```

Comment:
lets use return type of Promise<any> and not do this since this boolean value is meaningless